tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -233,4 +246,67 @@ public function data_maybe_redirect_to_onboarding() {
 			],
 		];
 	}
+
+	/**
+	 * Tests WC_Payments_Admin::add_disputes_notification_badge()
+	 */
+	public function test_disputes_notification_badge_display() {
+		global $submenu;
+
+		// Mock the database cache returning a set of disputes.
+		$this->mock_database_cache
+			->expects( $this->once() )
+			->method( 'get_or_add' )
+			->willReturn(
+				[
+					'needs_response'         => 1,
+					'warning_needs_response' => 3,
+					'won'                    => 2,
+					'lost'                   => 10,
+				]
+			);
+
+		$this->mock_current_user_is_admin();
+
+		// Make sure we render the menu with submenu items.
+		$this->mock_account->method( 'try_is_stripe_connected' )->willReturn( true );
+		$this->payments_admin->add_payments_menu();
+
+		$item_names_by_urls = wp_list_pluck( $submenu[ WC_Payments_Admin::PAYMENTS_SUBMENU_SLUG ], 0, 2 );
+		$dispute_menu_item  = $item_names_by_urls['wc-admin&path=/payments/disputes'];
```

Comment:
This test needs be updated when defining `$dispute_menu_item` as the path for the disputes page now includes the absolute wp-admin URL (`http://example.org/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fdisputes&filter=awaiting_response`).

```suggestion
		$dispute_menu_item  = array_values(
			array_filter(
				$item_names_by_urls,
				function( $key ) {
					return false !== strpos( $key, 'disputes' );
				},
				ARRAY_FILTER_USE_KEY
			)
		)[0];
```

Reply:
I decided to fix this in a slightly different way in 4d3480e567081ff33de1188c5b320ec62e6d14d0.